notebooks/MAST/m_dwarf_flares_hst/m_dwarf_flares_hst.ipynb

```diff
@@ -0,0 +1,1412 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    public_obs_tables = public_obs_tables[public_obs_tables['calib_level']==3]
</pre><p>+1 Julia. Explain why you want to use level 3. Also, this can be something you add to <code>.query_criteria</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>